lib/hyper/authentication_service/request.rb

```diff
@@ -27,6 +25,11 @@ def headers_for(user)
       rescue KeyError => e
         raise InvalidUserObjectError, e.message
       end
+
+      def handle_errors(response)
+        raise InternalServerError if response.status == 500
```

Comment:
@flanger001 are we concerned about other 5xx statuses?

Reply:
https://github.com/hyp3rventures/authentication_service/pull/7#discussion_r172287471